File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.cpp
#include <string>
#include <vector>

using namespace std;

bool solution(string s) {
    bool answer = true;
    if(s.length() != 4 && s.length() != 6) return false;
    for(char c : s) {
        if(c >= 'A') return false;
    }
    return answer;
}

File: 프로그래머스/1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
class Solution {
    public boolean solution(String s) {
        boolean answer = true;
        if(s.length() != 4 && s.length() != 6) return false;
        for(int i = 0; i < s.length(); i++) {
            char c = s.charAt(i);
            if(c < '0' || c > '9') {
                answer = false;
                break;
            }
        }
        return answer;
    }
}